### ISSUE TITLE
Android: Use captions, Caption text size and Caption styles

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30874]
## Requires API Proposal
This issue may require a new API, or a change to an existing API. An API proposal should be added and discussed before proceeding with implementation. The API proposal can be added in the comments of this issue or linked as a separate issue.

## Description
The AccessibilityInfo API supports getting the values of many system-level accessibility settings, but caption settings are not one of them.  Both iOS and Android have support for various caption-related settings, such as whether they are enabled by default, the text size and color, the text style, and the background color.

Building any experience in React Native that relies on audio being played should have a captioned alternative, and it should use these caption settings for the default display.

## React Native version:
v0.63

## Expected Behavior
The AccessibilityInfo API should have a way to query the various caption settings.

## Android Details
The CaptioningManager class (https://developer.android.com/reference/android/view/accessibility/CaptioningManager) is the way you can get access to the value of these settings.

## iOS Details
On iOS the Media Accessibility Framework (https://developer.apple.com/documentation/mediaaccessibility/) provides these values.

**Comment by blavalla** [comment]
While this is technically an issue, it's not very common that anyone would need to query this data in React Native, as it's unlikely they will be rendering captions themselves with RN, rather than in a video player itself.
